external/db_drivers/liblmdb/mdb_load.c

```diff
@@ -477,6 +477,10 @@ int main(int argc, char *argv[])
 					fprintf(stderr, "mdb_cursor_open failed, error %d %s\n", rc, mdb_strerror(rc));
 					goto txn_abort;
 				}
+				if (appflag & MDB_APPENDDUP) {
+					MDB_val k, d;
+					mdb_cursor_get(mc, &k, &d, MDB_LAST);
```

Comment:
Previous ops have all succeeded and the DB is non-empty, so there's no reason for this to fail. If some unexpected failure occurs it will be caught on the next loop.